src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/project.json

```diff
@@ -19,6 +19,8 @@
     "System.Threading": "4.4.0-beta-24619-02",
     "System.Threading.Thread": "4.4.0-beta-24619-02",
     "System.Threading.Tasks": "4.4.0-beta-24619-02",
+    "System.Xml.ReaderWriter": "4.4.0-beta-24619-02",
+    "System.Xml.XPath": "4.4.0-beta-24619-02",
```

Comment:
The reason I did not remove this project.json is in the [common tests' project.json](https://github.com/dotnet/corefx/blob/master/src/Common/test-runtime/project.json) `System.Runtime.Loader` does not have a run-time assembly for net463 on some platforms (win-x86, win-x64) supported there.